docs/UserGuide.adoc

```diff
@@ -296,6 +296,45 @@ Examples:
 * `import /Users/seedu/Documents/exportData.xml` +
 imports contact data to the location `/Users/seedu/Documents/exportData.xml`
 
+== Facebook Integration [Coming in v2.0]:
```

Comment:
This new heading along with its contents should be shifted below section:

> Synchronizing with the cloud [Coming in v2.0]:

because the following sections should not be under Facebook Integration:

> 3.4. Clearing all entries : clear
3.5. Exiting the program : exit
3.6. Saving the data
3.7. Synchronizing with the cloud [Coming in v2.0]: